README.md

```diff
@@ -244,6 +244,19 @@ In the example the current list `.sortable` allows items within it to be sorted
 
 If you want to be able to move items between to sortables, the `acceptFrom` option must be present on both of them.
```

Comment:
I know this is not you, but if you change it in any case, can you fix the typo
> move items between **to** sortables
to 
> move items between **two** sortables